src/ParametricOptInterface.jl

```diff
@@ -818,25 +843,100 @@ function MOI.optimize!(model::ParametricOptimizer)
         end
         empty!(model.updated_parameters)
     end
+
     MOI.optimize!(model.optimizer)
+
+    if model.evaluate_duals
+        calculate_dual_of_parameters(model)
+    end
 end
 
 ### Duals
 
-function MOI.get(model::ParametricOptimizer, attr::MOI.ConstraintDual, ci::MOI.ConstraintIndex)
-    return MOI.get(model.optimizer, attr, ci)
+function calculate_dual_of_parameters(model::ParametricOptimizer) 
+    param_dual_cum_sum = Dict{Int, Float64}()
+
+    for vi in keys(model.parameters)
+        param_dual_cum_sum[vi.value] = 0.0
+    end
+    
+    # TODO
+    # Quadratic constraints
+    for S in ScalarLinearSetList
+        affine_constraint_cache_inner = model.affine_constraint_cache[MOI.ScalarAffineFunction{Float64}, S{Float64}]
+
+        if length(affine_constraint_cache_inner) > 0
+            param_dual_cum_sum = iterate_over_constraint_cache_affine(model, affine_constraint_cache_inner ,param_dual_cum_sum)
+        end
```

Comment:
What about 
```
param_dual_cum_sum = iterate_over_constraint_cache_affine(model, model.affine_constraint_cache[MOI.ScalarAffineFunction{Float64}, S{Float64}] ,param_dual_cum_sum)
```

Reply:
Sometimes the dict `model.affine_constraint_cache[MOI.ScalarAffineFunction{Float64}, S{Float64}]` is empty, don't you think we could exploit that condition with a check like `if length(...) > 0` or `if !is_empty(...)` to continue the calculations?